.github/workflows/build-java-multi.yaml

```diff
@@ -0,0 +1,286 @@
+    needs: [ init, vulnerability-check-init ]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 10
+      matrix:
+        test: ${{ fromJSON(needs.vulnerability-check-init.outputs.imageList) }}
+    steps:
+      - id: init
+        name: Init Job
+        uses: hawk-ai-aml/github-actions/job-init@master
+        with:
+          slack-notification-webhook: ${{ secrets.CICD_MIGRATION_SLACK_WEBHOOK_URL }}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ORG_ECR_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_ORG_ECR_SECRET_ACCESS_KEY }}
+          aws-region: ${{ fromJson(needs.init.outputs.metadata).ecr.region }}
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+        with:
+          registries: ${{ fromJson(needs.init.outputs.metadata).ecr.registry }}
+
+      - name: Retrieve AWS ECR login password for Lacework
+        id: ecrLoginPassword
+        run: echo "loginPassword=$(aws ecr get-login-password --region eu-central-1)" >> $GITHUB_OUTPUT
+
+      - uses: lacework/lw-scanner-action@v1.3.1
+        name: Scan container image for vulnerabilities using Lacework
+        env:
+          JAVA_OFFLINE_MODE: ${{ env.LACEWORK_JAVA_OFFLINE_MODE }}
+        with:
+          LW_ACCOUNT_NAME: ${{ secrets.LACEWORK_ACCOUNT_NAME }}
+          LW_ACCESS_TOKEN: ${{ secrets.LACEWORK_ACCESS_TOKEN }}
+          IMAGE_NAME: ${{ matrix.test }}
+          IMAGE_TAG: ${{ env.HAWK_IMAGE_TAG }}
+          SAVE_RESULTS_IN_LACEWORK: true
+          ADDITIONAL_PARAMETERS: --docker-username AWS --docker-server https://860641649575.dkr.ecr.eu-central-1.amazonaws.com --docker-password ${{ steps.ecrLoginPassword.outputs.loginPassword }}
+
+  update-overlays:
+    needs: [ init, build ]
+    if: ${{ needs.init.outputs.update-kustomize == 'true' }}
+    runs-on: [ "self-hosted", "builder" ]
+    steps:
+      - id: init
+        name: Init Job
+        uses: hawk-ai-aml/github-actions/job-init@master
+        with:
+          slack-notification-webhook: ${{ secrets.CICD_MIGRATION_SLACK_WEBHOOK_URL }}
+
+      - id: update-overlay-github
+        name: Update kustomize overlay
+        if: fromJson(needs.init.outputs.parameters).kustomize == 'github' || fromJson(needs.init.outputs.parameters).kustomize == 'github-fake'
+        uses: hawk-ai-aml/github-actions/kustomize-overlay-multi@master
+        with:
+          component-name: ${{ inputs.component }}
+          component-modules: ${{ needs.init.outputs.kustomizable-modules }}
+          component-overlays: ${{ needs.init.outputs.overlays }}
+          component-tag: ${{ env.HAWK_IMAGE_TAG }}
+          kustomize-repo: ${{ fromJson(needs.init.outputs.parameters).kustomize == 'github-fake' && 'hawk-ai-aml/kustomize2.git' || 'hawk-ai-aml/kustomize.git' }}
```

Comment:
Are we simplifying all of these in a second moment? we don't need any of those github-fake, kustomize2 and many of these init variables.